ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/privacy/methods/priv/PrivGetTransactionReceipt.java

```diff
@@ -133,22 +134,22 @@ public JsonRpcResponse response(final JsonRpcRequestContext requestContext) {
 
     LOG.trace("Processed private transaction events");
 
-    final BytesValue transactionOutput =
+    final Bytes transactionOutput =
         privacyParameters
             .getPrivateStateStorage()
             .getTransactionOutput(txHash)
-            .orElse(BytesValue.wrap(new byte[0]));
+            .orElse(Bytes.wrap(new byte[0]));
 
-    final BytesValue revertReason =
+    final Bytes revertReason =
         privacyParameters.getPrivateStateStorage().getRevertReason(txHash).orElse(null);
 
     final String transactionStatus =
         Quantity.create(
-            BytesValues.asUnsignedBigInteger(
-                privacyParameters
-                    .getPrivateStateStorage()
-                    .getStatus(txHash)
-                    .orElse(BytesValue.wrap(new byte[0]))));
+            privacyParameters
+                .getPrivateStateStorage()
+                .getStatus(txHash)
+                .orElse(Bytes.wrap(new byte[0]))
```

Comment:
Should there be a Bytes.EMPTY value?  A whole lot less object churn for the GC.

Reply:
yep. Changed to use that.